### ISSUE TITLE
Speaker Verification Predict

### ISSUE THREAD

**Issue by Kailegh** [HarryVolek/PyTorch_Speaker_Verification#52]
Hi, first of all, thank you for your awesome work. I intend to use this for speaker verification, I know a speaker and I need to know if another audio belongs to him or not. I have some questions regarding your repo:  
- For the test, Why don't you use the speakers in training (the knowns ones) and test it against a set of unheard speakers?  As far as I have understood you split the test in two and use each half to predict against the other (kind of, because you randomize and end up using the whole test set, bu question remains)
- I had thought of saving the speaker embeddings, that is equivalent to the part of the code where you calculate the centroid right?  I could calculate the centroid of all my known speakers and just load it later.
- As I said my intention is to give the network and audio and test it against a known speaker, but I am a bit lost on how could I do that with your code, could you point me in the right direction? I get lost after you calculate the sim_matrix

Once more thank you for your work!
